src/App.vue

```diff
@@ -5,43 +5,27 @@
     <div class="wrap-outer-content layout-band">
       <div class="wrap-content">
         <div class="layout-band">
-          <SearchForm @searched="doSearch" />
+          <SearchForm @searched="receiveSearch" />
           <SearchMetadata
             v-if="showSearchMetadata"
             v-model:query.lazy="query"
             v-model:status="status"
             v-model:hits="hits"
           />
+          <div v-if="status.loading" class="panel panel-info">
```

Comment:
I think this block needs to be moved into the router view. You can see this not populating by searching for something like `asdfasdfasdf`